backend/spring-routie/src/main/java/routie/routiespace/service/RoutieSpaceService.java

```diff
@@ -0,0 +1,22 @@
+package routie.routiespace.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import routie.routiespace.domain.RoutieSpace;
+import routie.routiespace.domain.RoutieSpaceIdentifierProvider;
+import routie.routiespace.repository.RoutieSpaceRepository;
+
+@Service
+@RequiredArgsConstructor
+public class RoutieSpaceService {
+
+    private final RoutieSpaceRepository routieSpaceRepository;
+    private final RoutieSpaceIdentifierProvider routieSpaceIdentifierProvider;
+
+    @Transactional
+    public RoutieSpace add() {
```

Comment:
```suggestion
    public RoutieSpace addRoutieSpace() {
```
컨트롤러와 통일성이 필요할 것 같은데 어떻게 생각하시나요?